src/docfx/validation/ContentValidator.cs

```diff
@@ -49,7 +51,10 @@ internal class ContentValidator : ICollectionFactory
                 fileResolver.ResolveFilePath(_config.MarkdownValidationRules),
                 fileResolver.ResolveFilePath(_config.Allowlists),
                 fileResolver.ResolveFilePath(_config.SandboxEnabledModuleList),
-                this);
+                this,
+                EnvironmentVariable.GetDocsEnvironment().ToString(),
+                SecureScheme + _config.HostName,
```

Comment:
Why validator needs to understand `SecureScheme + _config.HostName`?

Reply:
This was suggested by Eason that he wanted the links validated by broken link lib are complete (https://xxx.com/xxx). To fulfil this, either docfx or docs.validation needs to add leading `https`. Since host name is from docfx, he suggested we did this in docfx as well.

Reply:
Is it possible to include the host name in `LinkNode` itself?

Reply:
resolved.